### ISSUE TITLE
glava: version information not available at command line

### ISSUE THREAD

**Issue by ryantm** [NixOS/nixpkgs#57483]
```
[nix-shell:~/.cache/nix-review/pr-57476]$ ./results/glava/bin/glava -V
GLava (glava) unknown (stable, unix/fhs)
```

_Originally posted by @ryantm in https://github.com/NixOS/nixpkgs/pull/57476#issuecomment-471846483_

cc @eadwu

**Comment by eadwu** [comment]
Seems like it is suppose to be unknown, since we don't keep the `.git` directory [1]. Though a simple patch otherwise.

[1] https://github.com/wacossusca34/glava/blob/master/Makefile#L74

**Comment by ryantm** [comment]
Closed by #57537
